src/main.ts

```diff
@@ -62,26 +64,28 @@ async function handleMain(inputs: Inputs, client: GitHubClient): Promise<void> {
   if (changes.additions.includes(inputs.haltFile)) {
     core.startGroup(`${inputs.defaultBranch}:${inputs.haltFile} added`);
     const msg = message.fromContent(
-      fs.readFileSync(inputs.haltFile).toString()
+      fs.readFileSync(inputs.haltFile).toString(),
     );
     core.info(`Halting all open PRs: ${msg.title}`);
     await haltOpenPullRequests(client, github.context, inputs, msg);
     await addWorkflowSummary(msg);
+    await sendSlackNotifications(inputs, msg);
     core.endGroup();
   }
 
   if (changes.removals.includes(inputs.haltFile)) {
     core.startGroup(`${inputs.defaultBranch}:${inputs.haltFile} removed`);
     core.info("Un-halting all open PRs");
     await unhaltOpenPullRequests(client, github.context, inputs);
+    await sendSlackNotifications(inputs);
     core.endGroup();
   }
 }
 
 async function handlePullRequest(
   inputs: Inputs,
   client: GitHubClient,
-  pullRequest: PullRequest
+  pullRequest: PullRequest,
```

Comment:
FYI: these are caused by prettier v2 -> v3. I think I isolated them in their own commit.